### DISCUSSION TITLE
Error viewing circuits in Liionpack {dependency error "pdflatex"}

### DISCUSSION THREAD

**Discussion by procrastinator10125** [pybamm-team/liionpack#314] [Q&A]
We need 'pdflatex' to view the circuit in liionpack, but there are other dependency error from 'attrs'

`
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
jsonschema 4.22.0 requires attrs>=22.2.0, but you have attrs 18.2.0 which is incompatible.
referencing 0.35.1 requires attrs>=22.2.0, but you have attrs 18.2.0 which is incompatible.
`

Also when I am upgrading the library 'attrs' to 22.2.0, the following error shows up

`
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
pdflatex 0.1.3 requires attrs<19.0,>=18.2, but you have attrs 22.2.0 which is incompatible.
`

**Comment by TomTranter** [comment]
I will probably remove thi splot functionality in the future in favour of something more pythonic

**Reply by RuiheLi** [reply]
Dear @TomTranter ,

in this case, is there any way to show graphically the circuit that is being solved? Thanks!

Kind regards,
Ruihe
